File: src/main/java/me/dolphago/domain/Following.java
package me.dolphago.domain;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

@ToString
@Entity
@NoArgsConstructor
@Getter
public class Following extends BaseEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String githubLogin;
    private String url;

    public Following(final String githubLogin, final String url) {
        this.githubLogin = githubLogin;
        this.url = url;
    }
}

File: src/main/kotlin/me/dolphago/domain/Following.kt
package me.dolphago.domain

import jakarta.persistence.*

@Entity
data class Following(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private val id: Long? = null,

    @Column(name = "github_login")
    val githubLogin: String,
    @Column(name = "url")
    val url: String
) : BaseEntity()